worker/deploy/worker.secure.js

```
@@ -0,0 +1,47 @@
+          status: 'ok',
+          timestamp: Date.now(),
+          relay: env.RELAY_URL,
+          version: '1.0.0',
+          hasToken: !!env.BOT_TOKEN  // Boolean only, never expose actual token
+        }), {
+          headers: { 'Content-Type': 'application/json' }
+        });
+        
+      case '/stats':
+        const lastPoll = await env.KV.get('last_poll_time');
+        return new Response(JSON.stringify({
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)